client_cli/src/main.rs

```diff
@@ -158,6 +156,7 @@ trait RunArgs {
     ///
     /// # Errors
     /// if inner command errors
+    // TODO: replace `eyre::Result` with `error_stack::Result`
```

Comment:
Please create ticket if not planned for current PR.

Reply:
I found it simpler to just remove this comment =)